Components/Snackbar/snackbar.css

```diff
@@ -0,0 +1,33 @@
+@import "../Buttons/buttons.css";
+@import "../../Variables/variables.css";
+
+.snackbar,
+.snackbar-center,
+.snackbar-right {
+  background-color: var(--black);
+  color: var(--white);
+  padding: 1rem;
+  width: 400px;
+  text-align: center;
+  border-radius: 5px;
+  bottom: 10px;
+  position: fixed;
```

Comment:
Are these common properties ? If yes, Please use common class name

Reply:
Yes, they're common but I was thinking from a modular point of view. if someone wants to import a right-sided snack bar, wouldn't be more convenient to use "snackbar-right" instead of "snackbar snackbar-right"?

Don't wanna have "snackbar right" as then "right" class from some other component can mess things up? Opinions?